core/assets/bundles/bundle_nl.properties

```diff
@@ -118,619 +134,654 @@ players = {0} Spelers online
+zone.fungalPass.name = Schimmelpad
+zone.groundZero.description = De optimale plek om weer tot kracht te komen. Weinig gevaar. Weinig grondstoffen.\nVerzamel zoveel mogelijk lood en koper als je kan.\nEn vertrek.
+zone.frozenForest.description = Ook hier, dicht bij de bergen, hebben de schrimmels zich verspreid. De koude tempratuur houd ze niet voor eeuwig tegen.\n\nBegin de industriele revolutie. Bouw fossiele generators. Leer hoe te repareren.
+zone.desertWastes.description = Deze woestijn is groot, onvoorspelbaar, en vol met oude technologie.\nSteenkool is hier te vinden. Verbrand het om stroom op te wekken, or verwerk het in grafiet.\n\n[lightgray]Of het hier veilig is is een tweede.
+zone.saltFlats.description = Aan de randen van de woestijn liggen de zoutvlaktes. Weinig grondstoffen zijn hier te vinden.\n\nDe vijand heeft hier rantsoenen opgeslagen. Vernietig hun core. Laat niks staan.
+zone.craters.description = Water heeft zich hier opgehoopt, herrinering aan de vroegere oorlog. Herrover dit gebied. Verzamel zand. Maak glas. Pomp water in je wapens en boren.
+zone.ruinousShores.description = Voorbij de ruines, is de kust. Lang geleden werd de kust hier verdedigd. Weinig ervan is terug te vinden. Enkel de meest simpele verdedigingswerken staan nog overeind, \nGa door met uitbereiden, herontdek de verloren techniek.
+zone.stainedMountains.description = Verder inland liggen de bergen, nog niet berijkt door schimmels.\nVerzamel de grote hoeveelheiden titanium in het gebied. Leer hoe je het kan gebruiken.\n\nDe vijand is krachtig hier. Geef ze geen tijd om je te overrompelen.
+zone.overgrowth.description = Dit gebied is overgroeid, dichter bij de bron van de schimmels.\nDe vijand heeft hier een uitkijkpost. Bouw dolk units. Vernietig hen. Claim was verloren was.
+zone.tarFields.description = De randen van een olie productie veld, tussen de bergen en de woestijn. Een van de weinige plekken met bruikbare olie.\nOndanks dat het verbannen is, zijn er wel krachtige vijanden in de buurt. Onderschat ze niet.\n\n[lightgray]Onderzoek wat je met olie kan doen indien mogelijk.
+zone.desolateRift.description = Een super gevaarlijk gebied. Veel grondstoffen, maar weinig ruimte. Grote kans op verwoesting. Lanceer zo snel mogelijk. Word niet overmoedig door de lange tijd tussen de rondes.
+zone.nuclearComplex.description = Een voormalige instalatie voor de productie en verwerking van thorium, ligt er nu verlaten bij.\n[lightgray]Onderzoek thorium en de vele toepassingen ervoor.\n\nDe vijand is aanwezig hier in grote getalen, constant aan het zoeken voor aanvallers.
+zone.fungalPass.description = Een transitiegebied tussen de hoger bergen en de lagergelegen, beschimmelde gebieden. Een kleine verkenningsbasis is hier gestationeerd.\nVernietig het.\nGebruik Dolk en Kruiper units. Maak de twee cores onbruikbaar.
```

Comment:
`hogere` or `hoger gelegen`, not a combination of the two.
Je kan menselijke patrouilles, ... stationeren. Je kan geen ding stationeren.